ngx_http_proxy_connect_module.c

```diff
@@ -832,7 +832,7 @@ ngx_http_proxy_connect_write_upstream(ngx_http_request_t *r,
 
     if (c->write->timedout) {
         ngx_connection_error(c, NGX_ETIMEDOUT,
-                             "upstream timed out (proxy_connect)");
+                             "upstream connection timed out (proxy_connect)");
```

Comment:
uncessary to fix this line. It refers to nginx upstream source code:

```
 $ grep 'upstream timed out' -R src
src/http/ngx_http_upstream.c:        ngx_connection_error(c, NGX_ETIMEDOUT, "upstream timed out");
src/http/ngx_http_upstream.c:        ngx_connection_error(c, NGX_ETIMEDOUT, "upstream timed out");
src/http/ngx_http_upstream.c:        ngx_connection_error(c, NGX_ETIMEDOUT, "upstream timed out");
```

Also note that it may be connecting timeout or sending timeout. We can distinguish this case with 'c->log->action', which is also logged by `ngx_connection_error()`

Reply:
hi @ylmrx

could u resolve this? Just drop this diff

Reply:
*done*